app/models/extract.rb

```diff
@@ -4,7 +4,6 @@ class Extract < ActiveRecord::Base
 
   validates_presence_of :quantity_value 
   validates_presence_of :quantity_unit
-  validates_presence_of :quantity_concentration 
   validates_presence_of :verbatim_anatomical_origin 
   validates_presence_of :year_made 
```

Comment:
Check the validation rules on collecting event start_ fields.  Dupe them over here for the date related validations.  Maybe write a spec for them.